### PR TITLE
Display connection problems in Banks

### PR DESCRIPTION
**Pull request by ptbrowne**
When one of the connection has a problem, we tell the user inside Banks so that they can fix that in the Home.

- Display connection status in Balance page
- Display more information and a button to the Home in Transaction page
  - The connection problem banner is displayed in function of the account filter; if we are seeing a revolut account, we will not have a banner signalling the the caisse d'epargne account has a problem
  - Since it is possible that groups contain one or more accounts with problems, we can display more than 1 error at a time. To reduce visual clutter, we display the errors in a carrousel.

image

image

### CODE REVIEW

File: src/components/effects.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { useEffect, useState } from 'react'
+import { Intents } from 'cozy-interapp'
+
+export const useRedirectionURL = (client, doctype, options) => {
+  const [fetched, setFetched] = useState(null)
+  useEffect(() => {
+    const fetch = async () => {
+      const intents = new Intents({ client: client })
+      const fetchedRes = await intents.getRedirectionURL(doctype, options)
```
Comment (drazik) [inline src/components/effects.js]: If it contains a redirectionURL, can we rename `fetched`, `setFetched` and `fetchedRes` to something like `redirectionURL`, `setRedirectionURL` and `fetchedRedirectionURL`?

File: src/ducks/balance/components/AccountRow.jsx [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import CozyClient, { queryConnect } from 'cozy-client'
 import { withBreakpoints, translate } from 'cozy-ui/react'
 import Icon from 'cozy-ui/react/Icon'
 import cx from 'classnames'
+import get from 'lodash/get'
```
Comment (drazik) [inline src/ducks/balance/components/AccountRow.jsx]: This can be grouped with the next line

File: src/ducks/balance/components/AccountRow.jsx [hunk 3]
```diff
@@ -16,32 +18,30 @@ import {
 import styles from 'ducks/balance/components/AccountRow.styl'
 import { HealthReimbursementsIcon } from 'ducks/balance/components/HealthReimbursementsIcon'
 import AccountIcon from 'components/AccountIcon'
+import { triggersConn } from 'doctypes'
+import { isErrored } from 'utils/triggers'
```
Comment (drazik) [inline src/ducks/balance/components/AccountRow.jsx]: Not part of this PR, but maybe we should extract these utils to cozy-doctypes

File: src/components/Carrousel.jsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    const {
+      children,
+      className,
+      breakpoints: { isMobile }
+    } = this.props
+    return (
+      <Media className={className}>
+        <Img
+          className={cx('u-c-pointer u-slateGrey', {
+            'u-o-0': currentIndex === 0
```
Comment (drazik) [inline src/components/Carrousel.jsx]: This doesn't prevent from clicking on the element. It is not visible but the user can still interact with it. If we want the element to keep its dimensions so nothing jumps, we can add `visibility: hidden` to it

File: src/components/Carrousel.jsx [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+            {range(children.length).map((x, i) => (
+              <span key={i} onClick={this.handleChange.bind(this, i)}>
+                {i === currentIndex ? '●' : '○'}
+              </span>
+            ))}
+          </div>
+        </Bd>
+        <Img
+          className={cx('u-c-pointer u-slateGrey', {
+            'u-o-0': currentIndex === children.length - 1
```
Comment (drazik) [inline src/components/Carrousel.jsx]: Same here with visibility

File: src/components/ErrorCard/index.jsx [hunk 6]
```diff
@@ -13,18 +13,23 @@ const ErrorCard = ({
   content,
   buttonLabel,
   buttonHref,
-  buttonIcon
+  buttonIcon,
+  style
 }) => {
   return (
-    <div className={styles.ErrorCard}>
+    <div className={styles.ErrorCard} style={style}>
```
Comment (drazik) [inline src/components/ErrorCard/index.jsx]: This can be done when extracting it to cozy-ui, but I think we should spread props, so the component can receive events, custom classname, etc

Comment (ptbrowne) [reply src/components/ErrorCard/index.jsx]: The ErrorCard will be removed in favor of using the Infos component from cozy-ui.

Will be done in https://github.com/cozy/cozy-banks/pull/1574

File: src/components/effects.js (hunk 1 above)
Comment (ptbrowne) [reply src/components/effects.js]: Done.

File: src/ducks/balance/components/AccountRow.jsx (hunk 2 above)
Comment (ptbrowne) [reply src/ducks/balance/components/AccountRow.jsx]: Done
